tests/components/brunt/test_config_flow.py

```diff
@@ -0,0 +1,196 @@
+"""Test the Brunt config flow."""
+from unittest.mock import patch
+
+from aiohttp import ClientResponseError
+from aiohttp.client_exceptions import ServerDisconnectedError
+
+from homeassistant import config_entries, data_entry_flow, setup
+from homeassistant.components.brunt.const import DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from tests.common import MockConfigEntry
+
+CONF = {CONF_USERNAME: "test-username", CONF_PASSWORD: "test-password"}
+
+
+async def _flow_submit(hass):
+    return await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": config_entries.SOURCE_USER},
+        data=CONF,
+    )
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] is None
+
+    with patch("brunt.BruntClientAsync.async_login", return_value=True), patch(
```

Comment:
Are we patching in the right place?

https://docs.python.org/3/library/unittest.mock.html#where-to-patch